app/coffee/Features/Project/ProjectController.coffee

```diff
@@ -139,6 +142,9 @@ module.exports = ProjectController =
 					return next(err)
 				logger.log results:results, user_id:user_id, "rendering project list"
 				tags = results.tags[0]
+				notifications = require("underscore").map results.notifications, (notification)-> 
```

Comment:
I should use require(underscore) instead of "_" reference because it get broken in the second call, I don't know why